apps/docsite/.storybook/theme.ts

```diff
@@ -0,0 +1,42 @@
+import { create } from '@storybook/theming';
```

Comment:
This component was copied over from https://github.com/microsoft/fluentui/blob/master/apps/public-docsite-v9/.storybook/theme.js and will be removed once original file will be published as a part of a new package.